cookbook/event_dispatcher/event_listener.rst

```diff
@@ -248,7 +248,7 @@ or a "sub request"::
 Certain things, like checking information on the *real* request, may not need to
 be done on the sub-request listeners.
 
-Events or Subscribers
+Listeners or Subscribers
 ---------------------
```

Comment:
And we should add a label for the old headline before the new one to keep the old anchor too:

```rst
.. _events-or-subscribers:
```